### DISCUSSION TITLE
Android - Submissions for EXPORT competition

### DISCUSSION THREAD

**Discussion by batrlatom** [ultralytics/yolov5#3434] [Show and tell]
## <div align="center">Compete and Win</div>

Use this discussion thread for YOLOv5 🚀 EXPORT Competition submissions in the **Android** category. Good luck!

<p align="center">
  <a href="https://github.com/ultralytics/yolov5/discussions/3213">
  <img width="850" src="https://github.com/ultralytics/yolov5/releases/download/v1.0/banner-export-competition.png"></a>
</p>

**Comment by lp6m** [comment]
Hello, please have a look.
https://github.com/lp6m/yolov5s_android

**Reply by batrlatom** [reply]
we will take a look. Thanks

**Reply by glenn-jocher** [reply]
@lp6m very impressive inference speeds on CPU! 

Thank you for your submission, @SergioSanchezUltralytics review it this month and we will make announcements on winners shortly thereafter.

**Reply by ghost** [reply]
@lp6m we are trying to reproduce your submission results for the YOLOv5 Export Competition and had a few questions:
1. We do not see the same inference+NMS times as your profiling results on the same Xiaomi Mi11 hardware.
2. Your app seems to be lacking a camera connection.

Are there any steps we are missing or code missing to complete the solution? Thanks!

**Reply by lp6m** [reply]
@SergioSanchezUltralytics Sorry for late reply.
1.   
The benchmarked latency listed in README.md is measured by TfLite Benchmark tool, and the latency does not contain preprocess and postprocess(NMS).  See [benchmark/README.md](https://github.com/lp6m/yolov5s_android/tree/master/benchmark) 
In order to maximize performance, we quit as many other applications as possible when benchmarking.
If you are unable to reproduce the results I reported, please provide details about the commands you used to run the benchmark, i.e., which delegates were enabled and which models you use to run the benchmark.
Unfortunately, I don't have Mi11 anymore, so I cannot perform benchmark again,  but the performance reported and the videos described in the README were measured and recorded on actual devices.

2.   
Please check your permission setting for the app. I have confirmed that the app in [release](https://github.com/lp6m/yolov5s_android/releases) works also in another phone like Mi11 Lite, Google Pixel3. (Of course fps decreases than running on Mi11.) 
In the README screenshot, the FPS of the camera function is 16.95, but this is a screenshot of the moment when the FPS is at its maximum. In reality, it varies in value from 9 to 16.

**Comment by stefani-kovachevska** [comment]
Hi @lp6m, could you please send me an email at [stefani@ultralytics.com](mailto:stefani@ultralytics.com) including your full name, so I can share more information with you? Thanks!

Stefani Kovachevska
_Project Manager at  Ultralytics_

**Reply by lp6m** [reply]
I sent an email from my gmail address.

**Comment by stefani-kovachevska** [comment]
Congratulations @lp6m for winning the prize in the Android category!
Check out the official announcement of the winners for each category of the Export competition here:  [https://ultralytics.com/article/export-competition-winners](https://ultralytics.com/article/export-competition-winners)

Stay amazing and keep creating! 🚀 
_Team Ultralytics_
